.gitignore

```diff
@@ -1,2 +1,3 @@
 *.swp
 mock/
+/rel-eng
```

Comment:
This was trying to sneak in...

Reply:
could you open a separate PR for this? Then I can cherry-pick it from develop to 1.x.